massa-execution-worker/Cargo.toml

```diff
@@ -23,10 +23,11 @@ tracing = "0.1"
 massa_execution_exports = { path = "../massa-execution-exports" }
 massa_models = { path = "../massa-models" }
 massa_hash = { path = "../massa-hash" }
-massa-sc-runtime = { git = "https://github.com/massalabs/massa-sc-runtime", tag = "v0.4.4" }
+massa-sc-runtime = { git = "https://github.com/massalabs/massa-sc-runtime", rev = "0719439066576b7c3681d9cf95c31722be6a4734" }
```

Comment:
Maybe add a todo or should be release a new version of massa sc runtime before merging this ?

Reply:
TODO was added, we'll see if we do end up releasing a new version of `massa-sc-runtime` before merging this.